exercises/largest-series-product/HINTS.md

```diff
@@ -0,0 +1,6 @@
+# Largest Series Product in Rust
+
+These iterators may be useful, depending on your approach
+
+- [Windows](https://doc.rust-lang.org/std/slice/struct.Windows.html)
+- [Product](https://doc.rust-lang.org/std/iter/trait.Product.html)
```

Comment:
I remember getting lost in the Rust documentation when I started learning the language, it is not trivial from the linked pages to get to the one with the examples. I would rather link the slice and iterator pages of the documentation here because it contains the descriptions of what these functions do and provides code examples.

  * https://doc.rust-lang.org/std/primitive.slice.html#method.windows 
  * https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.product